data/glib-2.0/schemas/com.github.cumulus.gschema.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <schemalist gettext-domain="cumulus">
-  <schema id="net.launchpad.cumulus" path="/net/launchpad/cumulus/">
+  <schema id="com.github.cumulus" path="/apps/cumulus/" gettext-domain="cumulus">
```

Comment:
need to remove one gettext-domain="cumulus" as it already was in schemalist, don't know how I missed it.